src/magicbox/SuggestionsManager.ts

```diff
@@ -232,7 +232,8 @@ export class SuggestionsManager {
     return $$('div', {
       className: this.suggestionListboxClassName,
       id: this.suggestionListboxID,
-      role: 'listbox'
+      role: 'listbox',
```

Comment:
Just to make sure: does every direct child of this element have `role="listitem"`?

Reply:
Yeah they do